src/resolve/resolved-manager.c

```diff
@@ -584,6 +584,7 @@ static void manager_set_defaults(Manager *m) {
         m->resolve_unicast_single_label = false;
         m->cache_from_localhost = false;
         m->stale_retention_usec = 0;
+        m->cache_size_per_scope = DEFAULT_CACHE_SIZE;
```

Comment:
Could you also update `manager_dispatch_reload_signal()` to make the new cache size is applied after reload ?